proto/weather_current.proto

```diff
@@ -0,0 +1,58 @@
+syntax = "proto3";
+package xbospb;
+import "nullabletypes.proto";
+message Weather_Current_State {
```

Comment:
Can we integrate this with the existing `Weather_Station` proto file instead of creating a new one? They should be easily unified and field should be named very similarly (especially since they are based on the same data source)